### PR TITLE
Module 1: Html and CSS Hello Microverse

### PR DESCRIPTION
**Pull request by alvinlouis29**
##The main objective of this project is to show some of the best practices working with Git and GitHub flow.
##In this project, I have built a simple webpage that displays the heading "Hello Microverse" in the color red.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello World</title>
+</head>
+<body>
+    <h1> Hello World</h1>
```
Comment (OmarRamoun) [inline index.html:11]: - [ ] [REQUIRED] Please, make sure to add a class to the `h1` tag and use it in the CSS as requested in the project requirements 🧾

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @alvinlouis29 👋

## Welcome to Microverse! 🌎

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights 🌟

- ✔️ very descriptive commit messages 
- ✔️ very descriptive PR title and description
- ✔️ no linter errors
- ⭐ Awesome README
- ⭐ Awesome implementation of project requirements
- The code is top-notch! 💯 and have a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (alvinlouis29) [reply index.html:11]: Thank you Mr Omar

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @alvinlouis29 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ descriptive commit messages 
- ✔️ descriptive PR title and description
- ✔️ no linter errors
- ✔️ implemented previous review requested changes
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**